### PR TITLE
drbd-utils: update to 9.12.2.

### PR DESCRIPTION
**Pull request by ailiop-git**

### CODE REVIEW

**Comment by ailiop-git** [comment]
builds fine under all archs, but the homepage is probably dropping repeated connections. Perhaps we need to make the distfile sticky during build? please merge.

**Comment by Chocimier** [comment]
9.13.1 is out

**Comment by ericonr** [comment]
Ci results are weird.

File: srcpkgs/drbd-utils/template [hunk 1]
```diff
@@ -26,13 +26,8 @@ short_desc="Distributed replicated storage"
...
 license="GPL-2.0-or-later"
 homepage="http://www.drbd.org"
-distfiles="https://www.linbit.com/downloads/drbd/utils/${pkgname}-${version}.tar.gz"
-checksum=1102e2a2001a45685c2f4ce5cb14e5a8a099044f53389d38480b3a88f5db3fd1
-
-# XXX mirror which is reachable from musl builders
-if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
-	distfiles="https://distfiles.voidlinux.de/${pkgname}-${version}.tar.gz"
-fi
+distfiles="https://github.com/LINBIT/${pkgname}/archive/v${version}.tar.gz"
```
Comment (ericonr) [inline srcpkgs/drbd-utils/template:29]:
```suggestion
distfiles="https://github.com/LINBIT/drdb-utils/archive/v${version}.tar.gz"
```

File: srcpkgs/drbd-utils/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'drbd-utils'
 pkgname=drbd-utils
-version=9.12.1
+version=9.13.1
```
Comment (ericonr) [inline srcpkgs/drbd-utils/template:3]: It seems 9.15.1 is out.

**Review by ericonr — COMMENTED** [review]
Please ping me once you're done with changes.

**Pull request closed without merging**